### PR TITLE
Migration file fixed

### PR DESCRIPTION
**Pull request by KerimCETINBAS**
fixed syntax issues, Persons table removed and its data merged with accounts table﻿ for sake of simplicity

### CODE REVIEW

File: migrations/20230915234055_create_user_table.sql [hunk 1]
```diff
@@ -1,39 +1,33 @@
...
-CREATE TABLE IF NOT EXISTS Persons (
-    id UUID DEFAULT uuid_generate_v4() NOT NULL,
-    fist_name VARCHAR(32) NOT NULL,
-    last_name VARCHAR(32) NOT NULL,
-    person_type PersonType NOT NULL DEFAULT PersonType.LEGAL
-    fk_account UUID NOT NULL,
-    PRIMARY KEY(id)
-    CONSTRAINT fk_account
-        FOREIGN KEY (fk_account) REFERENCES Accounts(ids)
-);
```
Comment (KerimCETINBAS) [inline migrations/20230915234055_create_user_table.sql:14]: Can be created on next migration

File: migrations/20230915234055_create_user_table.sql [hunk 2]
```diff
@@ -1,39 +1,33 @@
 -- +goose Up
 -- +goose StatementBegin
-CREATE Type PersonType AS ENUM ("LEGAL", "REAL")
+
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-CREATE TABLE IF NOT EXISTS Persons (
-    id UUID DEFAULT uuid_generate_v4() NOT NULL,
-    fist_name VARCHAR(32) NOT NULL,
-    last_name VARCHAR(32) NOT NULL,
-    person_type PersonType NOT NULL DEFAULT PersonType.LEGAL
```
Comment (caglarcercinli) [inline migrations/20230915234055_create_user_table.sql:9]: what do you mean with LEGAL? it is an enum, ok, the other option is REAL. but the difference is not clear.

Comment (KerimCETINBAS) [reply migrations/20230915234055_create_user_table.sql:9]: Legal persons present as business, corps, institutes etc. REAL persons basically  human

e.g in Turkish Gerçel ve Tüzel kişiler
https://en.wikipedia.org/wiki/Legal_person

Comment (KerimCETINBAS) [reply migrations/20230915234055_create_user_table.sql:9]: - caglarcercinli is a real user here
- edunota is a legal legal 
but both is an user

**Review by caglarcercinli — APPROVED** [review]

**Pull request merged**